src/components/HomePage/LearnMoreButton.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react'
+import styled from '@emotion/styled'
+import { space } from '@guardian/src-foundations'
+import { Link } from 'gatsby'
+import { Button, buttonBrandAlt } from '@guardian/src-button'
+import { ThemeProvider } from 'emotion-theming'
+import { SvgArrowRightStraight } from '@guardian/src-icons'
+
+const ButtonLink = styled(Link)`
+  margin-bottom: ${space[1]}px;
+`
```

Comment:
`Link` only works for internal links. Maybe we should be returning an error if `linkUrl` begins with `http`

Reply:
Good catch. Might make sense to use something different to the `Link` component given that it's Gatsby specific and has a particular intended use case